### ISSUE TITLE
Analytics KPI dashboard request for Caregiver Team (Health care application)

### ISSUE THREAD

**Issue by ala-yna** [department-of-veterans-affairs/va.gov-team#16547]
# Request a KPI Dashboard

## What this form is for
Use this template to request a KPI dashboard on Domo for your product.

## Instructions for the VFS team
- [x] For additional info, read through the [How to Request a KPI Dashboard Documentation](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/analytics/kpi-dashboard-request.md)
- [x] Your KPIs are clearly linked back to Veteran outcomes
- [x] Link your KPIs below (KPIs should be in your Product Outline)
- [x] Add your team label to follow progress

Is this a new KPI dashboard request? Or is this an iteration? Please check the corrrect response.
- [x] New dashboard
- [ ] Iteration of current dashboard

### KPIs

- How many Veterans are looking at the How to Apply page?
- How are Veterans getting to the How to Apply page?
- How many Veterans are completing and submitting the form online? 
- How does form completion compare to other forms on VA.gov?
- How many Veterans start the form but don’t complete it?
- Are there any points within the form where Veterans tend to get stuck?
- How are Veterans rating their experience using the form?
- How long does it take Veterans to fill out the online form?
- How many Veterans start filling out the form, save their progress, and then come back to it?
- What places in the form do Veterans tend to step away?
- How many Veterans don't complete the form after saving one in progress?
- Stretch:
     - Proportion of eligible veterans that apply online
     - Proportion of applications submitted online vs. on paper

*Please leave the following blank*

---

## VSP Analytics & Insights Acceptance Criteria

1. Groom Analytics
- [ ] Groom analytics for technical feasibility
    - [ ] Document data sources for each metric
    - [ ] Document unavailable vs. available metrics
- [ ] If any, add implementation issue for additional necessary implementation

2. Draft PST Dashboard
- [x] Draft dashboard on Domo
- [ ] Structure data for ETL
- [ ] Pipe available metrics to Domo
- [ ] Add appropriate team members to dashboard

Please see [checklist for full KPI dashboarding QA checklist](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsp/teams/insights-analytics/analytics-tools/domo-dashboard-qa-checklist.md)

## Definition of Done
- [ ] KPIs have been groomed and connected to Domo
- [ ] Dashboard has been shared back with the VFS team
- [ ] KPI Dashboard full QA checklist completed

**Comment by joanneesteban** [comment]
## Content & Usability Review

Thanks for pulling this together, @JasonTheBearded ! Looks good, just made some edits on the titles for consistency.

A couple other things for this dashboard as well:

- [ ] The filters between all the charts and the ForeSee chart aren't coordinating (when the ForeSee chart is filtered, the other charts don't filter, and vice-versa)
- [ ] For this metric: `How does form completion compare to other forms on VA.gov?` - low lift here might just be linking to the [Product Health MVP Google Sheet](https://docs.google.com/spreadsheets/d/1-1GYhR5RFItrYee2Yki_nuMW3XKYrWVzoZ8dppS3hb4/edit#gid=342453254)
- [ ] `How long does it take Veterans to fill out the online form?` -> do we have time to complete?
- [ ] For these:

> How many Veterans start filling out the form, save their progress, and then come back to it?
> How many Veterans don't complete the form after saving one in progress?

Are we able to look at funnels via Save in Progress?

**Comment by joanneesteban** [comment]
Thanks, @JasonTheBearded !

### Final To Do:
- [ ] Make a note on the ForeSee chart that there's an issue with filtering & to filter that chart as well if they're looking for additional filtering
- [ ] Is it possible to apply the dynamic date title to the funnel tables? When they're exported, it's unclear what the date range is for that data.
- [ ] Style sections with title headers. Wireframe below:
![image](https://user-images.githubusercontent.com/48964695/103937050-3b2e5080-50f6-11eb-8bc6-f2408fdb4ff0.png)
![image](https://user-images.githubusercontent.com/48964695/103937079-42edf500-50f6-11eb-8b9c-fb4843e217da.png)

Once this is done, we can set up an onboarding session with Alayna and Ming!

**Comment by JasonTheBearded** [comment]
@joanneesteban I finished all the requested changes. The only item I was not able to complete is the total time to complete line. Because the time field is a calculated field taking the average of all times for the step, DOMO does not total those times up but instead averages them. Attempting to use a Beast Mode calc to average the total time also results in an error.

**Comment by joanneesteban** [comment]
Thanks, Jason!

@ala-yna @mlighdc Can we set up a 15-30 min overview meeting in the next couple weeks to review this dashboard?

**Comment by ala-yna** [comment]
Certainly! I set up an invite for next Wednesday, let me know if there's a better time that works.

**Comment by joanneesteban** [comment]
Great! Next Wednesday sounds great - would 3:30-4pm EST work?

**Comment by joanneesteban** [comment]
WHOOPS. Sorry about that. 10:30 works 👍🏼

**Comment by joanneesteban** [comment]
### Feedback Items for closing:

- [ ] Completion rate over time
- [ ] Successful vs. unsuccessful submissions
- [ ] Clarifying the difference between successful vs. completion
- [ ] Fix the Product Health Sheet

**Comment by JasonTheBearded** [comment]
Hey @ala-yna I added the successful vs unsuccessful submissions viz to the dashboard. I'm working through the completion rate over time ask and wanted to get some additional clarification on what exactly that completion rate means to you. Thanks!

**Comment by JasonTheBearded** [comment]
Hey @ala-yna would you mind taking a look at my above comment? Thanks!

**Comment by ala-yna** [comment]
Hi @JasonTheBearded! Sorry for the delay! What I had in mind for the completion rate is wanting to understand this question: how many people start the form but _don't_ complete it?

**Comment by JasonTheBearded** [comment]
Thanks @ala-yna! So if you're wanting the inverse, are you wanting to see the number of people that start the form and _do_ complete it?

**Comment by JasonTheBearded** [comment]
@mandy-uxd Can you do a design review of this dash, https://va-gov.domo.com/page/980067552
Thanks!

**Comment by jonwehausen** [comment]
@JasonTheBearded & @mandy-uxd 

Please see [this ](https://github.com/department-of-veterans-affairs/va.gov-team/issues/18939)request from @ala-yna as an iteration to include a new page in the health care application funnel - `/veteran-home-address` page. This new page launched on 1/25 -- thanks in advance here.

**Comment by JasonTheBearded** [comment]
@ala-yna where in the application does that new page sit?

**Comment by ala-yna** [comment]
It is conditionally loaded after the va.gov/health-care/apply/application/veteran-information/veteran-address page. So, if a Veteran selects yes for "Is your home address the same as your mailing address?" it does not get loaded, and if they select no to that question, it becomes the next page of the form.

**Comment by ala-yna** [comment]
Also, @JasonTheBearded, one of our KPIs was "How long does it take Veterans to fill out the online form?"

Is that tracked somewhere in the dashboard/GA?

**Comment by JasonTheBearded** [comment]
Thanks @ala-yna!
And yes we are tracking that KPI. It's near the bottom of the dashboard, there's a section called 'How long is it taking Veterans to complete each step of the form?' and breaks down the time on page by step.

**Comment by mlighdc** [comment]
@JasonTheBearded for "How long it takes to complete?" Does it include the time when it is Save in Progress?

**Comment by JasonTheBearded** [comment]
@mlighdc can you provide some more details about what Save in Progress means? Are you looking for if we're tracking the time between pages loading or something different?

**Comment by mlighdc** [comment]
Hi @JasonTheBearded veterans can save their form so they can complete at a later time (Save in Progress - aka SiP). I was wondering how completion time is calculated or determined. Does it include the time the application is in "Save in Progress" state? I believe an application that is in Save in Progress is available to the veteran for 60 days, allowing them to come back and complete later.

Let me know if I can clarify more.

**Comment by JasonTheBearded** [comment]
@mlighdc Thanks for the clarification! In GA we're able to track time on page through a calculation checking the time difference between the entrance time of the page a veteran is currently on and the entrance time of the next page. GA doesn't give a whole lot more time tracking capability than that. The calculation is done on a day by day basis, so if a veteran saves, leaves, and comes back at a later date, there will be two time values for those two days. The dashboard is presenting the average time on page so we're looking at all values for that page across all days. In this instance we are not including the 'Save in Progress' time.

**Comment by mlighdc** [comment]
Sounds good @JasonTheBearded. That works. I was worry that it would include Save in Progress. On a related note, is there any way we can track % of people who has saved in progress and come to fully complete the application/form within 30/60 days?

**Comment by JasonTheBearded** [comment]
My team and I were actually just talking about that. We need to do a bit more discovery on it but it should be possible within our datasets.

**Comment by joanneesteban** [comment]
Notes for VSP: design needs to be broken down.

**Comment by mlighdc** [comment]
@joanneesteban - is your message for the VSP analytics team or the 10-10 team? Just want to clarify to make sure you are not waiting on something from us. Thanks!

**Comment by joanneesteban** [comment]
@mlighdc Sorry about that! That note was for our (VSP) team :D 

Also, is it okay if we close out this ticket & open a [new ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=joanneesteban&labels=analytics-insights%2C+analytics-request%2C+kpi-dashboard&template=analytics-request-kpi-dashboard.md&title=Analytics+KPI+dashboard+request+for+%5BTeam%5D) for this data discovery?

> is there any way we can track % of people who has saved in progress and come to fully complete the application/form within 30/60 days

**Comment by joanneesteban** [comment]
@mlighdc @ala-yna - we're closing this ticket on the Platform side. Please open a new ticket and we can reprioritize iterations for the dashboard!

**Comment by mlighdc** [comment]
@joanneesteban are you saying it is a different request to understand the % of people who completes the application after saving it would need a different ticket?

**Comment by joanneesteban** [comment]
@mlighdc yes, please! [A different ticket for an iteration/update](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=joanneesteban&labels=analytics-insights%2C+analytics-request%2C+kpi-dashboard&template=analytics-request-kpi-dashboard.md&title=Analytics+KPI+dashboard+request+for+%5BTeam%5D). The original scope of this ticket was to create the whole HCA dashboard, and given our limited resources, our team is trying to better gauge our customer support. A new ticket will help us move the % update into our queue as an update to the dashboard.

I apologize for the inconvenience, and appreciate your patience. We're trying to move through our support tickets, and part of that is being more precise about our open tickets so we can more accurately scope them.

**Comment by mlighdc** [comment]
okay @joanneesteban. I can work with @ala-yna to get a ticket started.

**Comment by joanneesteban** [comment]
@mlighdc thank you! Appreciate it.
